proposals/description/fip_70.ts

```diff
@@ -0,0 +1,52 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_70: ProposalDescription = {
+  title: 'FIP-70: ETH/FEI Liquidity migration to Balancer',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{balancerDepositFeiWeth}'],
+      description: 'Grant FEI/WETH Pool deposit the FEI_MINTER_ROLE role'
```

Comment:
future consideration, but we should stop granting minter to these contracts and just supply with FEI upfront